CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 # CHANGELOG
 
 ## [next]
+ - Never return `null` from `RotatingParticle.angle` getter
```

Comment:
I am not sure if we should have some much entries on the CHANGELOG for this PR, from the use perspective, it can be quite boring to read the changes in such details. I think we could an entry which captures the essence of this PR, something like: `- Code improvements and preparing APIs to null-safety`